test/Benchmarks/Exporter/OtlpLogExporterBenchmarks.cs

```diff
@@ -0,0 +1,158 @@
+using OpenTelemetryProtocol::OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
+using OpenTelemetryProtocol::OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.ExportClient;
+using OtlpCollector = OpenTelemetryProtocol::OpenTelemetry.Proto.Collector.Logs.V1;
+
+/*
+BenchmarkDotNet v0.13.6, Windows 11 (10.0.22621.2134/22H2/2022Update/SunValley2) (Hyper-V)
+AMD EPYC 7763, 1 CPU, 16 logical and 8 physical cores
+.NET SDK 7.0.400
+  [Host]     : .NET 7.0.10 (7.0.1023.36312), X64 RyuJIT AVX2
+  DefaultJob : .NET 7.0.10 (7.0.1023.36312), X64 RyuJIT AVX2
+
+
+|               Method |     Mean |   Error |  StdDev |   Gen0 |   Gen1 | Allocated |
+|--------------------- |---------:|--------:|--------:|-------:|-------:|----------:|
+| OtlpLogExporter_Http | 138.7 us | 2.08 us | 1.95 us | 0.4883 | 0.2441 |   9.85 KB |
+| OtlpLogExporter_Grpc | 268.3 us | 2.57 us | 2.28 us | 0.4883 |      - |   9.54 KB |
+*/
+
+namespace Benchmarks.Exporter;
+
+public class OtlpLogExporterBenchmarks
+{
+    private OtlpLogExporter exporter;
+    private LogRecord logRecord;
+    private CircularBuffer<LogRecord> logRecordBatch;
+
+    private IHost host;
+    private IDisposable server;
+    private string serverHost;
+    private int serverPort;
+
+    [GlobalSetup(Target = nameof(OtlpLogExporter_Grpc))]
+    public void GlobalSetupGrpc()
+    {
+        this.host = new HostBuilder()
+          .ConfigureWebHostDefaults(webBuilder => webBuilder
+               .ConfigureKestrel(options =>
+               {
+                   options.ListenLocalhost(4317, listenOptions => listenOptions.Protocols = Microsoft.AspNetCore.Server.Kestrel.Core.HttpProtocols.Http2);
+               })
+              .ConfigureServices(services =>
+              {
+                  services.AddGrpc();
+              })
+              .Configure(app =>
+              {
+                  app.UseRouting();
+                  app.UseEndpoints(endpoints =>
+                  {
+                      endpoints.MapGrpcService<MockLogService>();
+                  });
+              }))
+          .Start();
+
+        var options = new OtlpExporterOptions();
+        this.exporter = new OtlpLogExporter(options);
+
+        this.logRecord = LogRecordHelper.CreateTestLogRecord();
+        this.logRecordBatch = new CircularBuffer<LogRecord>(1);
+        this.logRecordBatch.Add(this.logRecord);
+    }
+
+    [GlobalSetup(Target = nameof(OtlpLogExporter_Http))]
+    public void GlobalSetupHttp()
+    {
+        this.server = TestHttpServer.RunServer(
+            (ctx) =>
+            {
+                ctx.Response.StatusCode = 200;
+                ctx.Response.OutputStream.Close();
+            },
+            out this.serverHost,
+            out this.serverPort);
+
+        var options = new OtlpExporterOptions
+        {
+            Endpoint = new Uri($"http://{this.serverHost}:{this.serverPort}"),
+        };
+        this.exporter = new OtlpLogExporter(
```

Comment:
Why use the internal overload of `OtlpLogExporter` ctor here? Could we be consistent with the Grpc benchmark and just use:
```csharp
var options = new OtlpExporterOptions
{
    Endpoint = new Uri($"http://{this.serverHost}:{this.serverPort}"),
};
this.exporter = new OtlpLogExporter(options);
```

Reply:
missed that. fixed it